.github/workflows/update-image-digest.yml

```
@@ -80,7 +80,7 @@ jobs:
           fi
 
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@923ad837f191474af6b1721408744feb989a4c27 # v4
+        uses: peter-evans/create-pull-request@v7
```

nit: let's pin again

```suggestion
        uses: peter-evans/create-pull-request@67ccf781d68cd99b580ae25a5c18a1cc84ffff1f #v7.0.6
```

---

Great point, thanks @filiptronicek , for the suggestion!

---

Still works: https://github.com/gitpod-io/gitpod/actions/runs/12636518855